website/sidebars.json

```diff
@@ -109,6 +109,7 @@
     ],
     "Maintain": [
       "maintain-index",
+	"maintain-polkadot-parameters",	
```

Comment:
The kusama-parameters markdown file also needs to be added inside /kusama-guide/sidebars.json so that it appears on the sidebar

Reply:
fixed in commit 104c719639cc65b125b01ad3ffd065ac8eb927bf